src/hooks/form/rules.ts

```diff
@@ -45,4 +45,13 @@ export const RULES: Record<RuleNameType, RuleType> = {
             return re.test(String(value));
         },
     },
+    // for min length, I am considering "a@a.in" as a valid mail
+    emailLength: {
+        type: "emailLength",
+        message: "Email length should be between 6 to 254 characters",
+        checkIsValid: (value: string) => {
+            const re = /^.{8,254}$/;
```

Comment:
I think there's a typo here, the `message` says the lower bound is 6, but in here it's 8.

Reply:
Resolved